code/datums/traits/good.dm

```diff
@@ -11,6 +11,23 @@
 
 
 
+/datum/trait/apathetic
```

Comment:
undocumented change (not that i mind the change beind made)

Reply:
```I also reordered the Jolly, Depression, and Apathetic traits to be alphabetical in the menu because it pleases my autism.```